release-handbook/major_release_workflow.md

```diff
@@ -0,0 +1,184 @@
+# How to manage doc for a major Zowe release
+
+Major Zowe releases follow a similar publication process to that of minor releases: content is staged, and then that content is published to the live Zowe Docs site.
+
+What is quite different about updating documentation for a major release is the amount and type of prep work required. To improve the chances of a smoother publication process, you will want to start preparing the doc work for a major release at least a year ahead of the major release GA date.
+
+Zowe major releases are typically done about every two years or so. Minor releases take place about once a calendar quarter.
+
+The publication process for minor releases is documented in the [Release handbook](release_handbook.md).
+
+For major Zowe releases, follow the process outlined here.
+
+## Major release publication process
+
+1. At least a year before the major release GA, Doc Squad works together to create a release strategy.
+    - Strategy to include:
+        - Schedule for reviewing existing content for any needed modifications
+        - Schedule for when modifications should be drafted, reviewed, and completed
+        - Complete inventory of Zowe Docs content and the people who are responsible for ensuring that content is ready for the major release.
+        - Assignment of responsibilities.
+2. Create a new branch based off the `master` branch to be dedicated to the major release documentation. Open the new branch in Visual Studio Code.
+    - For example, `v3-doc-branch` was used for the Zowe V3 release.
```

Comment:
Do we want these branches to be protected by GitHub?
This will prevent people accidentally pushing to the `v#-doc-branch`.